graph_altimeter/scan/iam/__init__.py

```diff
@@ -0,0 +1,309 @@
+"""This defines the logic that expands the IAM policies of a graph produced by
+Atimeter"""
+
+import json
+import os
+import re
+import uuid
+import logging
+
+from policyuniverse.policy import Policy
+from policyuniverse.arn import ARN
+
+from graph_altimeter import InvalidArnError, InvalidArnPattern
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
+
+# TODO: Strip the data from this file that is not needed.
+iam_definitions_path = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)),
+    "iam_definition.json"
+)
+
+
+def _load_iam_data(path):
+    """Builds a dictionary from the a conaining the information about the
+    iam resources and actions (for more information look at the README.md of
+    this directory). The keys of the dictionary are all the possible iam policy
+    actions, and the values are sets containing the resources they allow to
+    access. For instance: {'ec2:allocateaddres':{'elastic-ip',
+    'ipv4pool-ec2'}}"""
+    data = None
+    with open(path, "r") as file:
+        data = json.load(file)
+    actions = {}
+    for service in data:
+        prefix = service["prefix"]
+        for privilege in service["privileges"]:
+            action = privilege["privilege"].lower()
+            action = f"{prefix}:{action}"
+            resources = set()
+            for resource_type in privilege["resource_types"]:
+                if "resource_type" not in resource_type:
+                    continue
+                resource = resource_type["resource_type"].replace("*", "")
+                if resource == "":
+                    continue
+                # There is an exception with names of the resources, for s3
+                # buckets, there are some actions that report they apply to the
+                # resource 'object' but we need to call them 'bucket' instead.
```

Comment:
Can you reword this comment to explain why we need to call them `bucket`?

Reply:
👍